Gemfile.jruby-2.5.lock.release

```diff
@@ -213,7 +212,7 @@ GEM
       elasticsearch (>= 5.0.3)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       manticore (~> 0.6)
-    logstash-filter-fingerprint (3.2.1)
+    logstash-filter-fingerprint (3.2.2)
```

Comment:
`3.2.2` includes a fix for consistent firgerprint in hash https://github.com/logstash-plugins/logstash-filter-fingerprint/pull/55